src/main/java/org/openjdk/shenandoah/ShenandoahVisualizer.java

```diff
@@ -672,8 +687,23 @@ public synchronized void run() {
                 snapshot = cur;
                 lastSnapshots.add(new SnapshotView(cur));
                 popupSnapshots.add(cur);
-                if (lastSnapshots.size() > graphWidth / STEP_X) {
+
+                if (lastSnapshots.size() > (graphWidth - 50) / STEP_X) {
```

Comment:
Seems we compute this quantity `(graphWidth - 50) / STEP_X` quite a lot. Could we put it in a variable with a meaningful name?

Reply:
Okay that sounds good I'll think of a name for this variable.